### ISSUE TITLE
Food Bug Eat and Drop Fix

### ISSUE THREAD

**Issue by Fluriax** [Imanity-Software/ImanitySpigot3-git#264]
### Is your feature request related to a problem?

Because Players can abuse the Food Bug Glitch when they eat and drop the food they dont get the eat animation and instant fooded

### Describe the solution you'd like.

https://medal.tv/games/minecraft/clips/jfUEfhejs8C4kH2ht?invite=cr-MSxHUWMsMjkwNzIxNzExLA

### Describe alternatives you've considered.

I turned on the FoodBug fix but it dont work fine. ive seen other spigots they working

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have ensured the feature I'm requesting isn't already in the latest ImanitySpigot3.

**Comment by Xefreh** [comment]
hi, can you try to disable mouseDelayFix to see if the eatAndDropFix option is still not working as expected please. also can you explain the steps on how to reproduce the bug thanks.

**Comment by Fluriax** [comment]
its not working. i disabled mousedelayfix and enable eatdropfix still not working when i run drop eat

**Comment by Xefreh** [comment]
yes the bug happens because when the sprint packet is sent, the server doesn't release the item.

**Comment by Fluriax** [comment]
sure.
